CMakeLists.txt

```diff
@@ -943,6 +945,7 @@ list(APPEND SRC_FILES
     ${PROJECT_SOURCE_DIR}/src/mbgl/util/mat4.cpp
     ${PROJECT_SOURCE_DIR}/src/mbgl/util/mat4.hpp
     ${PROJECT_SOURCE_DIR}/src/mbgl/util/math.hpp
+    ${PROJECT_SOURCE_DIR}/src/mbgl/util/polyline_generator.cpp
```

Comment:
Thanks for factoring out this code!

Question: we have quite a bit of different modules in `util` already. Would adding it to another existing module or creating a new module/directory be a better fit?

Reply:
yeah, I thought about it, maybe `include/mbgl/gfx` and `src/mbgl/gfx` would be better? Since it's platform agnostic graphics code...

Reply:
Yes sounds good!